subprojects/docs/src/docs/userguide/dep-man/03-controlling-transitive-dependencies/platforms.adoc

```diff
@@ -63,3 +414,43 @@ You can enable this behavior by using the `enforcedPlatform` keyword, instead of
 include::sample[dir="snippets/dependencyManagement/managingTransitiveDependencies-enforcedConstraintsFromBOM/kotlin",files="build.gradle.kts[tags=dependency-on-bom]"]
 ====
+
+[[sub:platforms-vs-catalog]]
+== Should I use a platform or a catalog?
+
+Because platforms and catalogs both talk about dependency versions, can both be used to share dependency versions in a project, there might be a confusion regarding what to use and if one is preferable to the other.
+
+Ignoring the incubating state of version catalogs, the answer is that you should always use catalogs and _maybe_ use platforms in addition.
+
+A catalog helps with centralizing the dependency versions and is only, as it name implies, a catalog of dependencies you can pick from.
+We recommend using it to declare the coordinates of your dependencies, in all cases.
+It will be used by Gradle to generate type-safe accessors, present short-hand notations for external dependencies and it allows sharing easily those coordinates between different projects.
+Using a catalog will not have any kind of consequence on downstream consumers: it's transparent to them.
+
+A platform is a more heavyweight construct: it's a component of a dependency graph, like any other library.
+If you depend on a platform, that platform is itself a component in the graph.
+It means, in particular, that:
+
+- <<dependency_constraints.adoc#dependency-constraints,constraints>> defined in a platform can influence _transitive_ dependencies, not only the direct dependencies of your project
+- a platform is versioned, and a transitive dependency in the graph can depend on a different version of the platform, causing various dependency upgrades
+- a platform can tie components together, and in particular can be used as a construct for <<dependency_version_alignment.adoc#version_alignment, aligning versions>>
+- a dependency on a platform is "inherited" by the consumers of your dependency: it means that a dependency on a platform can influence what versions of libraries would be used by your consumers even if you don't directly, or transitively, depend on components the platform references
+
+In summary, using a catalog is always a good engineering practice as it centralizes common definitions, allows sharing of dependency versions or plugin versions, but it is an "implementation detail" of the build: it will not be visible to consumers and unused elements of a catalog are just ignored.
+
+A platform is meant to influence the dependency resolution graph, for example by adding constraints on transitive dependencies: it's a solution for structuring a dependency graph and influencing the resolution result.
+
+In practice, your project can both use a catalog _and_ declare a platform which itself uses the catalog:
+
+```
+plugins {
+    id 'java-platform'
+}
+
+dependencies {
+    constraints {
+        api(libs.mylib) // a platform can reference a dependency declared in a catalog
+    }
+}
+```
```

Comment:
Ideally this should be a snippet. If that's too complicated here, please provide pth Groovy and Kotlin DSL inline.